components/script/dom/bindings/error.rs

```diff
@@ -341,10 +275,19 @@ pub(crate) fn throw_constructor_without_new(cx: SafeJSContext, name: &str) {
     unsafe { throw_type_error(*cx, &error) };
 }
 
-impl Error {
+pub(crate) trait ErrorToJsval {
+    fn to_jsval(
+        self,
+        cx: SafeJSContext,
+        global: &GlobalScope,
+        rval: MutableHandleValue,
+    );
+}
```

Comment:
We should be able to remove this when more migration is done, right?

Reply:
It will be possible to move the to_jsval impl into script_bindings once the migration is done.